components/conversations/SmallComment.tsx

```diff
@@ -0,0 +1,119 @@
+import RichTextEditor from '../RichTextEditor';
+
+import CommentActions from './CommentActions';
+import { editCommentMutation, mutationOptions } from './graphql';
+import type { CommentProps } from './types';
+
+export default function SmallComment(props: CommentProps) {
+  const [isEditing, setEditing] = React.useState(false);
+  const hasActions = !isEditing;
+  const comment = props.comment;
+  const anchorHash = `comment-${new Date(comment.createdAt).getTime()}`;
+  const isPrivateNote = comment.type === commentTypes.PRIVATE_NOTE;
+
+  return (
+    <div
+      className="relative w-full border-slate-200 py-[16px] first:border-none first:pt-0 [&:last-child_.timeline-indicator]:bottom-[-16px] [&:last-child_.timeline-separator]:hidden"
+      data-cy="comment"
+      id={anchorHash}
+    >
+      <div className="timeline-separator absolute bottom-0 left-[20px] right-0 border-b" />
+      <div className="flex justify-between">
+        <div className="flex gap-4">
+          <div className="relative">
+            <div
+              className={clsx('timeline-indicator absolute bottom-[-16px] left-[20px] top-[-16px] border-l', {
+                'border-blue-400': isPrivateNote,
+              })}
+            />
+            <AccountHoverCard
+              account={comment.fromAccount}
+              trigger={
+                <div className="relative">
+                  <Avatar collective={comment.fromAccount} radius={40} />
+                  {isPrivateNote && (
+                    <div className="absolute bottom-[-4px] right-[-4px] flex h-[20px] w-[20px] items-center justify-center rounded-full bg-white shadow">
+                      <Lock size={16} className="text-blue-400" />
+                    </div>
+                  )}
+                </div>
+              }
+            />
+          </div>
+          <div>
+            <div className="mb-1 text-sm font-medium leading-5">{comment.fromAccount.name}</div>
+            <div className="text-sm leading-4 text-[#75777A]">
+              <DateTime dateStyle="medium" value={comment.createdAt} />
+            </div>
+            <div className="mt-4">
+              <InlineEditField
+                mutation={editCommentMutation}
+                mutationOptions={mutationOptions}
+                values={comment}
+                field="html"
+                canEdit={props.canEdit}
+                canDelete={props.canDelete}
+                isEditing={isEditing}
+                showEditIcon={false}
+                prepareVariables={(comment, html) => ({ comment: { id: comment.id, html } })}
+                disableEditor={() => setEditing(false)}
+                warnIfUnsavedChanges
+                required
+              >
+                {({ isEditing, setValue, setUploading }) =>
+                  !isEditing ? (
+                    <HTMLContent
+                      fontSize="13px"
+                      maxCollapsedHeight={40}
```

Comment:
This collapse limit seems a bit aggressive collapsing fairly small comments:

| Current (40) | Suggestion (140) |
|--------|--------|
| <img width="750" alt="Screenshot 2024-09-04 at 12 14 04" src="https://github.com/user-attachments/assets/5c9a3250-e2e0-4ba0-a1ac-24874c188cd1"> | <img width="754" alt="Screenshot 2024-09-04 at 12 12 31" src="https://github.com/user-attachments/assets/c13915bd-93f6-4535-8c9f-90f92e2b6f8f"> |

```suggestion
                      maxCollapsedHeight={140}
```